src/Analyzers/CSharp/CodeFixes/UseImplicitOrExplicitType/UseExplicitTypeCodeFixProvider.cs

```diff
@@ -96,12 +104,29 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             {
                 typeSyntax = typeSyntax.StripRefIfNeeded();
 
+                var newTypeSymbol = semanticModel.GetTypeInfo(typeSyntax, cancellationToken).ConvertedType;
+
+                if (newTypeSymbol.NullableAnnotation == NullableAnnotation.Annotated && variableIdentifier.HasValue)
```

Comment:
Are we confident that `newTypeSymbol` will be non-null here? The `ConvertedType` member is annotated for `null` returns hence this code will one day warn on this access.

Reply:
I'll try and bite the bullet and enable null annotations on this file to fix that :)